programs/openbook-twap/src/lib.rs

```diff
@@ -693,8 +699,8 @@ pub enum OpenBookTWAPError {
         "The `close_market_admin` of the underlying market must be equal to the `TWAPMarket` PDA"
     )]
     InvalidCloseMarketAdmin,
-    #[msg("Market must not expire (have `time_expiry` == 0)")]
-    NonZeroExpiry,
+    #[msg("Market must expire a minimum of 10 days from now")]
+    InsufficentExpiryTime,
```

Comment:
I guess someone could create a market that expired too long, but they'd only hurt themselves

Reply:
Yeh, it could definitely be too long. That's their problem. The default in the script should probably be 11 days or something.